src/pages/service/ResearchToggle.jsx

```diff
@@ -0,0 +1,141 @@
+import { useState, useRef } from 'react';
+import { downArrowIcon, researchImage, rightArrowIcon } from '../../assets';
+import { NavLink } from 'react-router-dom';
+import PropTypes from 'prop-types';
+
+const ToggleableSection = ({ title }) => {
```

Comment:
don't use this syntax use suggested one
```suggestion
const ToggleableSection = (props) => {
const { title } = props
```